src/Admin/SiteHealth.php

```diff
@@ -156,12 +157,47 @@ private function get_persistent_object_cache_learn_more_action() {
 	public function persistent_object_cache() {
 		$is_using_object_cache = wp_using_ext_object_cache();
 
+		$description = '<p>' . __( 'The AMP plugin performs at its best when persistent object cache is enabled. Object caching is used to more effectively store image dimensions and parsed CSS.', 'amp' ) . '</p>';
+
+		if ( empty( $is_using_object_cache ) ) {
+			$available_cache = $this->check_available_cache();
+
+			/* translators: plugin recommendation markup */
+			$plugin_placeholder = _x( 'We recommend to use %s plugin.', 'plugin recommendation markup', 'amp' );
```

Comment:
We may want to not recommend any specific plugin, but instead point them to documentation that may guide them how to configure object caching for their site. As mentioned in the issue by @westonruter:

> Instead of mentioning the specific plugins in the Site Health info recommendation, the list of backend-specific plugins can be in a documentation page on amp-wp.org. This documentation page can pull together the various pieces of information about object caching spread out across WordPress.org, including:
> 
> * https://make.wordpress.org/hosting/handbook/handbook/performance/#object-cache
> * https://developer.wordpress.org/reference/classes/wp_object_cache/#persistent-caching
> 
> Ultimately, the user should ask their host for the recommended way to enable object caching first before they do it themselves. So our recommendation should be to first contact the host, and then continue with our guidance to try the plugins we've identified which may work with their site.
> 
> See support topics about object caching:
> 
> * https://wordpress.org/support/topic/how-enable-persistent-object-in-amp-plugin/
> * https://wordpress.org/support/topic/amp-error-25/
> * https://wordpress.org/support/topic/persistent-object-cache-is-not-enabled-2/
> * https://wordpress.org/support/topic/persistent-object-caching-is-not-enabled-2/
> * https://wordpress.org/support/topic/persistent-object-cache-is-not-enabled-amp/
> 
> Anything we can do to make it easier for users to enable object caching the better.

Reply:
What about linking off to search results for object caching plugins using a given backend? For example:

* https://wordpress.org/plugins/search/redis+object+cache/
* https://wordpress.org/plugins/search/memcached+object+cache/
* https://wordpress.org/plugins/search/apcu+object+cache/

And then we can include with that a link to our object caching guide so they can see which plugins specifically we've tested: https://amp-wp.org/documentation/getting-started/amp-site-setup/persistent-object-caching/#i-need-help-setting-up-my-persistent-object-cache

At the very lest we can just say: “It looks like you have X available on your server.” where X is the backend service and then they will provide them with a starting point for which plugins to look for.